docs/content/users/topics/sharing.md

```diff
@@ -46,7 +46,7 @@ This set of steps assumes an ngrok subdomain `mg2`:
 * Run `bin/magento setup:store-config:set --base-url="https://mg2.ngrok.io/`.
 * Run [`ddev share`](../usage/commands.md#share) and you’ll see your project at `mg2.ngrok.io`.
 
-### Using nip.io or Custom Name Resolution on Your Local Network
+## Using nip.io or Custom Name Resolution on Your Local Network
```

Comment:
I realize you’ve just fixed the heading level here, but is it okay to squeeze in an edit for brevity? (I checked and there aren’t any links to this that need updating.)

```suggestion
## Using nip.io or Custom Name Resolution Locally
```